src/test/helper.js

```diff
@@ -11,6 +11,9 @@ if (fs.existsSync(".env.test")) {
 import sinon from "sinon"
 global.sinon = sinon
 
+import Bluebird from "bluebird"
+global.Promise = Bluebird
```

Comment:
Wait how come we're using Bluebird for testing and not native promises?

Reply:
We do use Bluebird in the app, but [it’s setup](https://github.com/artsy/metaphysics/blob/4968b86e19bd38acb823d2cacdbf5bcd1b7688ba/index.js#L21) in a module that’s not loaded in tests.